packages/syntax/src/languages.ts

```diff
@@ -48,6 +48,7 @@ export const languageParsers = {
 
 export const SupportedLanguages = {
   JavaScript: 'javascript',
+  JS: 'js', // Javascript
```

Comment:
is it worth adding `ts` while we're at it?

Reply:
Done :)